blogposts/sourcegraph-2-12-release-notes.md

```diff
@@ -0,0 +1,70 @@
+---
+title:  'Sourcegraph 2.12: Build your own Sourcegraph extensions, GitLab integration, code discussions, and self-service checkout'
+author: 'Dan Adler'
+publishDate: 2018-10-15T12:00-07:00
+tags: [
+  "blog"
+]
+slug: sourcegraph-2-12-release-notes
+heroImage: //images.ctfassets.net/le3mxztn6yoo/t4Qpcq5kA0AYM24Ws4mOk/4edf5502a936bbec90c262fa00355aed/sourcegraph-mark.png
+published: true
+---
+
+We are building [the best code search and browsing tool for your team](/) to help you write, review, and ship code better. Our mission is to [bring the future sooner](/plan) by bringing the best tools to developers everywhere.
+
+Our announcement this month that [Sourcegraph is now open source](/blog/sourcegraph-is-now-open-source/) is one way we will get there even faster.
+
+Here’s new, what’s changed, and what’s fixed in Sourcegraph 2.12.
+
+- [Build and publish your own Sourcegraph extensions](/blog/extension-authoring) (like the [Codecov Sourcegraph extension](https://twitter.com/srcgraph/status/1038200208138502144))
+
+- [Sourcegraph for GitLab](/blog/sourcegraph-for-gitlab)
+
+- [Discuss code and docs in repositories](/blog/discuss-code-and-docs-in-repositories)
+
+- [Product subscriptions, self-service checkout, and easy upgrades to Enterprise](/blog/product-subscriptions-and-license-keys)
+
+- [New product packaging and pricing](/blog/pricing-and-package-changes)
+
+- [Full Changelog (from 2.10 to 2.11)](#full-changelog-from-211-to-212)
+
+Ready to install or upgrade? [Get Sourcegraph for yourself (to get started)](/) or [deploy Sourcegraph for your team](/docs).
+
+## Full Changelog (from 2.11 to 2.12)
```

Comment:
@attfarhan I assume you synced this with the latest? My version was from last week.

Reply:
Yep